### PR TITLE
i18n(it): translated docs/guides/backend/xata.mdx

### PR DESCRIPTION
**Pull request by vclemenzi**
#### What kind of changes does this PR include?

- Translated content

#### Description

translated `docs/guides/backend/xata.mdx`

### CODE REVIEW

File: src/content/docs/it/guides/backend/xata.mdx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  readonly XATA_FALLBACK_BRANCH?: string;
+}
+
+interface ImportMeta {
+  readonly env: ImportMetaEnv;
+}
+```
+
+:::tip
+Per ulteriori informazioni sulle [variabili d'ambiente](/it/guides/environment-variables/) e sui file `.env` in Astro, leggi di più.
```
Comment (csarnataro) [inline src/content/docs/it/guides/backend/xata.mdx:60]: "leggi di più" at the end of the sentence doesn't sound right to me. Maybe "Leggi questa guida per avere ulteriori informazioni [...]"?
What do you think?

Comment (vclemenzi) [reply src/content/docs/it/guides/backend/xata.mdx:60]: I agree

Comment (vclemenzi) [reply src/content/docs/it/guides/backend/xata.mdx:60]: I agree

**Review by csarnataro — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**